flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/AbstractTableEnvironment.scala

```diff
@@ -83,4 +84,17 @@ class AbstractTableEnvironment {
     )
     TranslationContext.registerTable(dataSetTable, name)
   }
+
+  /**
+   * Execute a SQL query on a batch [[Table]].
```

Comment:
Keep the description of this method more generic as it will be the entry point for stream SQL queries as well.